stories/icons.stories.mdx

```diff
@@ -0,0 +1,445 @@
+    <td>(not to mix with sort-down)</td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-search"></i></td>
+    <td>7</td>
+    <td>Button/action</td>
+    <td>-</td>
+    <td>Search</td>
+    <td>FA</td>
+    <td>Search</td>
+    <td><i class="fas fa-search"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td>In the search bar</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-ellipsis-v"></i></td>
+    <td>8</td>
+    <td>Button/action</td>
+    <td>-</td>
+    <td>More/additional actions</td>
+    <td>FA</td>
+    <td>Ellipsis-v</td>
+    <td><i class="fas fa-ellipsis-v"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-question-circle"></i></td>
+    <td>9</td>
+    <td>Button/action</td>
+    <td>-</td>
+    <td>Info</td>
+    <td>FA</td>
+    <td>Question-circle</td>
+    <td><i class="fas fa-question-circle"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td>For tooltips</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-sync"></i></td>
+    <td>10</td>
+    <td>Button/action</td>
+    <td>-</td>
+    <td>Sync</td>
+    <td>FA</td>
+    <td>Sync</td>
+    <td><i class="fas fa-sync"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td>refreshing the Metrics, next to the time selector</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-file-export"></i></td>
+    <td>11</td>
+    <td>Button/action</td>
+    <td>-</td>
+    <td>Export</td>
+    <td>FA</td>
+    <td>File-export</td>
+    <td><i class="fas fa-file-export"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="far fa-clone"></i></td>
+    <td>12</td>
+    <td>Button/action</td>
+    <td></td>
+    <td>Copy</td>
+    <td>FA</td>
+    <td>Clone</td>
+    <td>&lt;i class="far fa-clone"&gt;&lt;/i&gt;</td>
+    <td>regular Style (far)</td>
+    <td>yes</td>
+    <td>small icon next to a field that could be copied</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-sort"></i></td>
+    <td>1</td>
+    <td>Table</td>
+    <td>Sorting</td>
+    <td>Sort</td>
+    <td>FA</td>
+    <td>Sort</td>
+    <td><i class="fas fa-sort"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-sort-up"></i></td>
+    <td>2</td>
+    <td>Table</td>
+    <td>Sorting</td>
+    <td>Sort-up</td>
+    <td>FA</td>
+    <td>Sort-up</td>
+    <td><i class="fas fa-sort-up"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i class="fas fa-sort-down"></i></td>
+    <td>3</td>
+    <td>Table</td>
+    <td>Sorting</td>
+    <td>Sort-down</td>
+    <td>FA</td>
+    <td>Sort-down</td>
+    <td><i class="fas fa-sort-down"></i></td>
+    <td>solid Style (fas)</td>
+    <td>yes</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
```

Comment:
Not sure to understand this last row.

Reply:
Agree, not clear: it's not a real icon. 
In the case of a cell with no value in a table, we use a simple "hyphen" (`&#45;`) from the font, for still displaying there is no value, without having too much visual noise.
As it's still useful to know about this specific use case, this info could be added in the "Table" component/guideline.
![image](https://user-images.githubusercontent.com/28654802/109863749-4d45ee80-7c62-11eb-979a-ca3ebbd4e989.png)